.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-Fixes #
+## Fixes #
```

Comment:
Does this effect GitHub's ability to [auto-link/close the associated issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)?

Reply:
I've put this on my latest PRs, we can test with those

Reply:
I tested in this one too, so we're good! :) https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/2133